### ISSUE TITLE
Main Nav - Left Panel needs a close button

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#189]
Normal way to exit out of it is to click off the side panel on the grayed out original page.

For google TalkBack tabbing/swiping through content, cannot target outside of the panel to close it. Add a physical real close option.
